### PR TITLE
Fix some compilation issues.

### PR DESCRIPTION
**Pull request by llyzs**

### CODE REVIEW

**Comment by hardening** [comment]
Hum can FreeRDP really work without any timer support ?

**Comment by llyzs** [comment]
> Hum can FreeRDP really work without any timer support ?

"CreateWaitableTimer" search result shows that it is used only in client implementation, and libfreerdp does not use this function. It looks like the timer implementation in winpr is just a convenience for client implementation but not a required component.

We are currently investigating the possibility to use freerdp in WebAssembly and the wasm environment provides no such implementation.

File: winpr/libwinpr/crypto/cipher.c [hunk 1]
```diff
@@ -93,7 +93,7 @@ static WINPR_RC4_CTX* winpr_RC4_New_Internal(const BYTE* key, size_t keylen, BOO
 	if (!evp)
 		goto fail;
 
-	EVP_CIPHER_CTX_init(ctx->ctx);
+	EVP_CIPHER_CTX_reset(ctx->ctx);
```
Comment (akallabeth) [inline winpr/libwinpr/crypto/cipher.c:96]: this is openssl >= 3.0 only, needs a preprocessor switch

**Review by akallabeth — CHANGES_REQUESTED** [review]

**Review by akallabeth — APPROVED** [review]
Sorry, did miss it was already part of 1.1.1
only 1.0 is lacking this, so LGTM

**Pull request merged**